CustomDynamicFont/Sources/Presentation/Modules/iOS12SwiftUI/DynamicFontSwiftUIView.swift

```diff
@@ -5,29 +5,90 @@
 //  Created by Bliss on 29/11/21.
 //  Copyright © 2021 Nimble. All rights reserved.
 //
+// swiftlint:disable closure_body_length
 
+import Combine
 import SwiftUI
 
 @available(iOS 13.0.0, *)
 struct DynamicFontSwiftUIView: View {
     
+    @State private var fontSliderValue: Double = 1.0
+    @State private var selectedOverrideOption = 0
+    @State private var fontSize: ContentSizeCategory?
+    private let overrideFontSizes: [ContentSizeCategory] = [.small, .medium, .large, .extraLarge]
+
     var body: some View {
         ScrollView {
             VStack(alignment: .center, spacing: 8.0) {
                 Image(uiImage: R.image.color_Rectangle() ?? UIImage())
                     .resizable()
                     .frame(width: 260.0, height: 260.0)
-                    .padding(.bottom, 12.0)
+                Spacer(minLength: 4.0)
                 Text(UIFont.ZenOldMincho.regular.fontName())
-                    .scaledFont(font: UIFont.ZenOldMincho.bold, forTextStyle: .headline)
+                    .scaledFont(font: UIFont.ZenOldMincho.bold, forTextStyle: .headline, overrideFontSize: fontSize)
                 Text(R.string.localizable.osVersionIos13Title())
-                    .scaledFont(font: UIFont.ZenOldMincho.regular, forTextStyle: .body)
+                    .scaledFont(font: UIFont.ZenOldMincho.regular, forTextStyle: .body, overrideFontSize: fontSize)
                 Text(R.string.localizable.ios12SwiftUILifecycleLabelTitle())
-                    .scaledFont(font: UIFont.ZenOldMincho.regular, forTextStyle: .body)
+                    .scaledFont(font: UIFont.ZenOldMincho.regular, forTextStyle: .body, overrideFontSize: fontSize)
                 Text(R.string.localizable.ios11UIKitCommentLabelTitle())
-                    .scaledFont(font: UIFont.ZenOldMincho.regular, forTextStyle: .footnote)
+                    .scaledFont(font: UIFont.ZenOldMincho.regular, forTextStyle: .footnote, overrideFontSize: fontSize)
+                Spacer(minLength: 4.0)
+
+                HStack {
+                    VStack(alignment: .leading, spacing: 8.0) {
+                        Text(R.string.localizable.ios11UIKitOverrideFontScaleTitleTitle())
+                            .scaledFont(
+                                font: UIFont.ZenOldMincho.regular,
+                                forTextStyle: .body,
+                                overrideFontSize: fontSize
+                            )
+                        Picker(
+                            selection: $selectedOverrideOption,
+                            content: {
+                                Text(R.string.localizable.ios11UIKitOverrideFontSegmentOptionsOff()).tag(0)
+                                Text(R.string.localizable.ios11UIKitOverrideFontSegmentOptionsOn()).tag(1)
```

Comment:
Can we define private constants for these tags so that they can represent some meaning when reading the code, instead of just pure number 0 and 1? Then we can reuse the constants on where the tags value is used as well. What do you think?